NineChronicles/2023-10/PlanetSpec.schema.json

```
@@ -57,6 +57,27 @@
                         "format": "uri"
                     }
                 },
+                "world-boss.gql": {
```

As I understood, the world-boss-service provides REST API. (https://github.com/planetarium/world-boss-service/blob/main/world_boss/app/api.py#L40) cc @ipdae 

```suggestion
                "world-boss.rest": {
```